src/types.ts

```diff
@@ -134,13 +134,13 @@ export interface VideoPlayerProps extends VideoProperties {
    * StyleSheet passed to the <Video /> component
    *
    */
-  videoStyle?: ViewStyle;
+   videoStyle?: StyleProp<ViewStyle>;
 
   /**
    * Container styles
    *
    */
-  containerStyle?: ViewStyle;
+   containerStyle?: StyleProps<ViewStyle>;
```

Comment:
There's a typo: it should be `StyleProp`, not `StyleProps` and it should be imported from `react-native`.

```javascript
import type {StyleProp} from 'react-native';
```

Reply:
@kumaryogesh17 Should be `StyleProp` for  `containerStyle` as well.

Reply:
I suggest you to fork the repo locally and run `yarn install`. That way the IDE will let you know about any errors before pushing. 👌

Reply:
Since you also have some eslint/prettier issues now and you won't be able to see them in the web browser :(

Reply:
![Screenshot from 2022-09-09 21-55-32](https://user-images.githubusercontent.com/82561880/189397064-b00c83ec-e312-4646-9e3d-f82649c1783d.png)

Reply:
🙌  That looks good so far, but now we have some `eslint`/`prettier` issues: 

<img width="673" alt="image" src="https://user-images.githubusercontent.com/55203625/189399306-3c9e2e62-c8b8-4619-9faa-12e69be53928.png">

Try enabling `eslint` and `prettier` in your editor. You can search about that on Google.

Reply:
okay

Reply:
More precisely, to solve the first error you'll have to add a space on line 2 before `StyleProp`.

For the second and third error, you'll have to delete the space BEFORE `videoStyle` and `containerStyle` I believe.

But by enabling `eslint` and `prettier` in your editor, they should be fixed automatically by `CTRL + S`